tests/test_binexport_features.py

```diff
@@ -0,0 +1,458 @@
+# Copyright (C) 2024 Mandiant, Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at: [package root]/LICENSE.txt
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+#  is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import binascii
+
+import pytest
+import fixtures
+
+import capa.features.file
+import capa.features.insn
+import capa.features.common
+import capa.features.basicblock
+from capa.features.common import (
+    OS,
+    OS_LINUX,
+    ARCH_I386,
+    FORMAT_PE,
+    ARCH_AMD64,
+    FORMAT_ELF,
+    OS_ANDROID,
+    OS_WINDOWS,
+    ARCH_AARCH64,
+    Arch,
+    Format,
+)
+
+FEATURE_PRESENCE_TESTS_BE2_ELF_AARCH64 = sorted(
+    [
+        # file/string
+        (
+            "687e79.be2",
+            "file",
+            capa.features.common.String("AppDataService start"),
+            True,
+        ),
+        ("687e79.be2", "file", capa.features.common.String("nope"), False),
+        # file/sections
+        ("687e79.be2", "file", capa.features.file.Section(".text"), True),
+        ("687e79.be2", "file", capa.features.file.Section(".nope"), False),
+        # file/exports
+        (
+            "687e79.be2",
+            "file",
+            capa.features.file.Export("android::clearDir"),
+            "xfail: not implemented yet?!",
+        ),
```

Comment:
We don't handle name demangling (see #2112)

Reply:
good, will update the reference in the code - I don't think this is an important feature for now

Reply:
💯